### PR TITLE
Update React Quickstart to mention scopes on the API

### PR DESCRIPTION
**Pull request by frederikprijck**
The React Quickstart mentions using custom scopes but has no reference to configuring these in your API with Auth0.
I added a small info box that references https://auth0.com/docs/get-started/dashboard/api-settings

### CODE REVIEW

File: articles/quickstart/spa/react/02-calling-an-api.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ ReactDOM.render(
 );
 ```
 
+:::note
+As Auth0 is only able to issue Tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```
Comment (stevehobbsdev) [inline articles/quickstart/spa/react/02-calling-an-api.md]:
```suggestion
As Auth0 is only able to issue tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Looks good, just a very minor thing

**Review by stevehobbsdev — DISMISSED** [review]

File: articles/quickstart/spa/react/02-calling-an-api.md [hunk 2]
```diff
@@ -46,6 +46,10 @@ ReactDOM.render(
 );
 ```
 
+:::note
+As Auth0 is only able to issue tokens for custom scopes that exist on your API, ensure the scopes used above are defined when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```
Comment (DanOnCall) [inline articles/quickstart/spa/react/02-calling-an-api.md]:
```suggestion
As Auth0 can only issue tokens for custom scopes that exist on your API, ensure that you define the scopes used above when [setting up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
```

I would reword it as the above using the active voice :)

**Review by DanOnCall — CHANGES_REQUESTED** [review]

**Review by DanOnCall — APPROVED** [review]
